cypress/support/modals.po.ts

```diff
@@ -26,4 +41,93 @@ export class ModalsPo extends BaseComponent {
     directiveEvents: 'demo-modal-events',
     directiveAutoShow: 'demo-modal-auto-shown'
   };
+
+  // temporary placed here
```

Comment:
?

Reply:
the method will be removed from here after the `e2e for typeahead` merged. the method is placed in `base.components`